parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -1239,7 +1239,6 @@ private static void processCryptoMetaData(FileMetaData fileMetaData, InputStream
           throw new IOException("Applying decryptor on plaintext file");
         }
       }
-      return; 
```

Comment:
No changes are needed in this file. Reading plaintext files with non-null decryptor is already enabled in the latest encr branch code.